Assignment/css/style.css

```diff
@@ -0,0 +1,202 @@
+     clear: both;
+     visibility: hidden;
+     line-height: 0;
+     height: 0;
+}
+
+.clear {
+  clear: both;
+}
+
+.alignright {
+  float: right;
+  padding: 0 0 10px 10px; /* note the padding around a right floated image */
+}
+
+.alignleft {
+  float: left;
+  padding: 0 10px 10px 0; /* note the padding around a left floated image */
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+body {
+    color: #000;
+    font-size: 12px;
+    line-height: 1.4;
+    font-family: avenir, sans-serif, helvetica;
+}
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+#container1 {
+  width: 100%;
+  min-width: 75em;
+  height: 43.75em;
+  background: url(../images/header_bg.jpg) #868D9D no-repeat center top;
+  background-size: cover;
+}
+
+header {
+  margin: 0em 5em;
+}
+
+section.benefits {
+  height: 30em;
+  background-color: #F2F6F5;
+  padding: 5em;
+}
+
+section.testimonial {
+  height: 20em;
+  background: url(../images/testimonial_bg.jpg) #051022 no-repeat center top;
+  padding: 5em;
+}
+
+.signupnow-sec {
+  width: 100%;
+  background-color: #F4F416;
+  height: 8em;
+}
+
+footer {
+  width: 100%;
+  height: 20em;
+  background-color: black;
+}
+
+/******************************************
+/* ADDITIONAL STYLES
+/*******************************************/
+
+h1, h2 {
+  color: white;
+}
+
+header h1 {
+  font-family: verdana, arial;
+  font-size: 3em;
+  font-weight: bold;
+  padding-top: 0.5em;
+  padding-left: 1.5em;
+}
+
+header h2 {
+  font-family: verdana, arial;
+  font-size: 2.5em;
+  margin-top: 3em;
+  margin-left: 15%;
+  white-space: nowrap;
+}
+
+.benefits {
+  color: #919B9C;
+  font-size: 1.25em;
+}
+
+.benefits p {
+  width: 24em;
+}
+
+h2.benefits {
+  color: #01214B;
+  font-size: 2.5em;
+}
+
+.benefits-list {
+  list-style: none;
+  margin: 0em;
+  padding: 1.5em;
+  font-size: 1.25em;
+  width: 20em;
+}
+
+.benefits-list li {
+  margin: 0.25em;
+}
+
+.benefits-list li:before {
+  content: "• ";
+  color: #4A5A5A;
+}
+
+.testimonial, .feedback-person {
+  text-align: center;
+  color: white;
+  font-size: 1.75em;
+  font-style: italic;
+}
+
+.feedback-person {
+  font-weight: bold;
+  font-style: normal;
+}
+
+footer {
+  color: white;
+  text-align: center;
+  padding-top: 7em;
+}
+
+footer h1 {
+  color: yellow;
+  font-size: 3em;
+  font-weight: bold;
+}
+
+/*********************************
+/* BUTTONS
+/*********************************/
+
+.getnow-button {
+  color: black;
+  background-color: yellow;
+  font-size: 1.5em;
+  font-weight: bold;
+  text-align: center;
+  padding: 0.3em;
+  width: 10em;
+  margin-top: 4em;
+  margin-left: 40%;
+  border-radius: 0.25em;
+}
+
+.signup-button {
+  color: white;
+  background-color: #01214C;
+  font-size: 1.5em;
+  text-align: center;
+  padding: 0.3em;
+  width: 10em;
+  margin-top: 0.75em;
+  margin-left: 40%;
```

Comment:
Use

```
margin-left: auto;
margin-right: auto;
```

To center instead.

Using 40% only works on certain screen sizes since the button is a fixed value.  If your button is also 20% of the screen then using 40% would work.